styleguide/.npmignore

```diff
@@ -12,6 +12,7 @@ build/components.html
 build/docs.html
 build/index.html
 scripts
+aXeReports
```

Comment:
aXeReports or axeReports? ;P

Reply:
aXe is how it's written ;) honestly though either way is fine by me :P